app/plugins/Files/js/loaders.js

```diff
@@ -76,7 +89,9 @@ function uploadDir(dirPath, nickname) {
 					notify('Cannot read ' + file, 'error');
 					return;
 				}
-				if (!isUnixHiddenPath(filePath) && stats.isFile()) {
+				if (file.slice(-4) === '.sia') {
+					loadDotSia(filePath);
```

Comment:
What if one wants to upload a directory of .sia files though? Should it be a separate option like with uploading a single file?